doc/articles/uno-development/contributing-intro.md

```diff
@@ -22,7 +22,7 @@ For the prerequisites you'll need, as well as useful tips like using [solution f
 
 If you're doing development for Uno's macOS support, you'll need to build and run Uno using Visual Studio for Mac. [There's a separate guide for that here](building-uno-macos.md).
 
-You can contribute to Uno directly from your browser using GitPod. [Find out how.](../features/working-with-gitpod.md)
+You can contribute to Uno directly from your browser using GitPod, [find out how](../features/working-with-gitpod.md).
 
 Whether you're fixing a bug or working on a new feature, [inspecting the visual tree of a running app](debugging-inspect-visual-tree.md) is often a key step.
```

Comment:
`[inspecting the visual tree of a running app](debugging-inspect-visual-tree.md)`
@morning4coffe-dev can we use xref: instead of .md file here please